### PR TITLE
converting badges to html

### PR DESCRIPTION
**Pull request by aleivag**
to show proper links

### CODE REVIEW

**Comment by jaymzh** [comment]
They're already links? This is non-standard, why?

**Comment by davide125** [comment]
Right now they're links to images, this makes them links to... useful things, hopefully.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,17 @@
 pystemd
 =======
 
-![Continuous Integration](https://github.com/systemd/pystemd/workflows/Continuous%20Integration/badge.svg?event=push) ![Matrix](https://img.shields.io/matrix/pystemd:matrix.org)
+<div align="center">
+  <p>
+    <a href="https://github.com/systemd/pystemd/actions/workflows/ci.yml">
```
Comment (davide125) [inline README.md:6]: This should probably link to https://github.com/systemd/pystemd/actions ?

Comment (jaymzh) [reply README.md:6]: The proper syntax is:
```
[![Continuous Integration](https://github.com/systemd/pystemd/actions/workflows/ci.yml/badge.svg)](https://github.com/systemd/pystemd/actions/workflows/ci.yml)
```

That will be a link and a badge image.

Then do the same thing with the matrix thing

Comment (aleivag) [reply README.md:6]: yeap... @jaymzh  its right... updating the PR in a sec (just saw the actuall systemd README page)

**Review by jaymzh — APPROVED** [review]

**Review by davide125 — APPROVED** [review]

**Pull request merged**